packages/react/src/views/ChatBody/ChatBody.js

```diff
@@ -204,6 +231,42 @@ const ChatBody = ({
 
   return (
     <>
+      {channelInfo.announcement && showAnnouncement && (
+        <Box css={styles.announcementStyles}>
+          <Box
+            ref={announcementRef}
+            css={[
+              styles.announcementTextBox,
+              css`
+                text-decoration: ${isOverflowing ? 'underline' : 'none'};
```

Comment:
Why adding a underline when overflowing ? Specific reason ?

Reply:
Just to give a hint to user that the announcement is clickable , but I just realized that in RC the underline appears only if the user hovers over the announcement , should I consider removing it in normal view?

Reply:
Yaa better remove it, keep it on hover only

Reply:
I removed the text underline in normal view